### PR TITLE
plugins: added Datadog::Notifications::Plugins::GRPC

### PR DESCRIPTION
**Pull request by mxmCherry**
For gRPC service instrumentation (for [grpcx](https://github.com/bsm/grpcx) gem reporting).

Looks extremely simple, so I did not bother with tests.

Bumped minor version (new backward-compatible functionality).

### CODE REVIEW

File: lib/datadog/notifications/plugins.rb [hunk 1]
```diff
@@ -5,6 +5,6 @@ module Plugins
   end
 end
 
-%w[base grape active_record active_job action_controller].each do |name|
+%w[base grape active_record active_job action_controller grpc].each do |name|
```
Comment (dim) [inline lib/datadog/notifications/plugins.rb]: instead, can we do this:

```
 module Datadog
   class Notifications
     module Plugins
       autoload :Base, 'datadog/notifications/plugins/base'
       autoload :ActiveRecord, 'datadog/notifications/plugins/active_record'
       ...
     end
  end
end
```

File: lib/datadog/notifications/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Datadog
   class Notifications
-    VERSION = '0.5.1'.freeze
+    VERSION = '0.6.0'.freeze
```
Comment (dim) [inline lib/datadog/notifications/version.rb]: 0.5.2 is fine, no need for major change

Comment (mxmCherry) [reply lib/datadog/notifications/version.rb]: @dim well, yes, given this is `0.x.x`, but I followed: https://semver.org/

> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.

File: lib/datadog/notifications/plugins.rb (hunk 1 above)
Comment (mxmCherry) [reply lib/datadog/notifications/plugins.rb]: Btw, why `autoload :Name, 'path/to/file'`? Looks like everything's put in right place so should be resolved just fine, no?

File: lib/datadog/notifications/version.rb (hunk 2 above)
Comment (mxmCherry) [reply lib/datadog/notifications/version.rb]: @dim will change if you insist

File: Gemfile.lock [hunk 3]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    datadog-notifications (0.5.1)
+    datadog-notifications (0.6.0)
```
Comment (dim) [inline Gemfile.lock]: run bundle again pls

File: lib/datadog/notifications/version.rb (hunk 2 above)
Comment (dim) [reply lib/datadog/notifications/version.rb]: yes, there is no breaking change here. with versions <1 semver is different. MINOR version are for breaking changes

**Review by dim — APPROVED** [review]

**Pull request merged**